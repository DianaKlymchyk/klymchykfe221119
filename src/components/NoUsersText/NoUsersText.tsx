import React, {FC} from 'react';
import './NoUsersText.css';

interface Props {
    text: string;
}

const NoUsersText: FC<Props> = ({text}) => {
    const onClickHandler = () => {
        console.log('on click on no text div')
    }

    return (
        <div className="no-users" onClick={onClickHandler}>{text}</div>
    );
}

export default NoUsersText;
