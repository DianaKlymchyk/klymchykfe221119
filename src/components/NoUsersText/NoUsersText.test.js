import renderer from 'react-test-renderer';
import NoUsersText from "./NoUsersText";

it('no users text render', () => {
  const component = renderer.create(
    <NoUsersText text="NoUsersText"/>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

  renderer.act(() => {
    tree.props.onClick();
  });

  tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});