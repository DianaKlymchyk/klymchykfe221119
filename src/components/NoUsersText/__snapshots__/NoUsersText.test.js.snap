// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`no users text render 1`] = `
<div
  className="no-users"
  onClick={[Function]}
>
  NoUsersText
</div>
`;

exports[`no users text render 2`] = `
<div
  className="no-users"
  onClick={[Function]}
>
  NoUsersText
</div>
`;
